jubakit/test/test_classifier.py

```diff
@@ -86,6 +86,49 @@ def test_invalid_get_labels(self):
     # when actually iterating over it, pass it to list().
     self.assertRaises(RuntimeError, list, ds.get_labels())
 
+  def test_from(self):
```

Comment:
I think it is better to name it `test_from_data` as it tests `from_data` method.